src/legacy/core_plugins/kibana/public/management/sections/objects/lib/resolve_saved_objects.js

```diff
@@ -79,24 +79,25 @@ async function importIndexPattern(doc, indexPatterns, overwriteAll, confirmModal
   let newId = await emptyPattern.create(overwriteAll);
   if (!newId) {
     // We can override and we want to prompt for confirmation
-    try {
-      await confirmModalPromise(
-        i18n.translate('kbn.management.indexPattern.confirmOverwriteLabel', {
-          values: { title: this.title },
-          defaultMessage: "Are you sure you want to overwrite '{title}'?",
+    const isConfirmed = await confirmModalPromise(
```

Comment:
Why is this still using confirmModalPromise?

Reply:
This `confirmModalPromise` is set up here:
https://github.com/elastic/kibana/blob/b4d3c3cb82f70944773ba5c09b2e1dac00bbbe22/src/legacy/core_plugins/kibana/public/management/sections/objects/_objects.js#L56

It's still from a new platform, I just decided not to rename the parameter.

`confirmModalPromise` is passed through `ObjectsTable` -> `Flyout` -> `resolveSavedObjects` -> `importIndexPattern`.